packages/itwin/tree-widget/src/components/trees/VisibilityTreeRenderer.tsx

```diff
@@ -30,18 +61,24 @@ export const useVisibilityTreeRenderer = (iconsEnabled: boolean, descriptionsEna
 };
 
 const imageLoader = new TreeImageLoader();
+
 /**
  * Creates node renderer which renders node with eye checkbox.
  * @public
  */
-export const createVisibilityTreeNodeRenderer = (iconsEnabled: boolean, descriptionEnabled: boolean) => {
-  return (props: TreeNodeRendererProps) => ( // eslint-disable-line react/display-name
+export const createVisibilityTreeNodeRenderer = ({ levelOffset = 20, expansionToggleWidth = 24, disableRootNodeCollapse = false, ...props }: visibilityTreeRendererProps) => {
+  return (treeNodeProps: TreeNodeRendererProps) => ( // eslint-disable-line react/display-name
```

Comment:
`return function VisibilityTreeNodeRenderer(treeNodeProps: TreeNodeRendererProps) {` should resolve disabled lint error

Reply:
fixed